cpp/open3d/t/geometry/RaycastingScene.cpp

```diff
@@ -689,16 +1169,50 @@ struct RaycastingScene::Impl {
                     LoopFn);
         }
     }
+
```

Comment:
Can we use STL here for ArraySum and ArrayPartialSum, and memcpy for CopyArray?

Reply:
b2689cbe1d0b26c0cccef23bc355d3429bbd4692